egs/wsj/s5/steps/nnet3/xconfig_to_configs.py

```diff
@@ -170,6 +185,12 @@ def write_config_files(config_dir, all_layers):
             # preserves the backtrace
             raise
 
+    # remove previous init.config
```

Comment:
the init.config is the switcher of LDA computing, previous version should remove anyway.